### PR TITLE
chore(example-notebooks): update notebook images with latest versions of packages

### LINKED ISSUE #7313: Update all notebook server images
**Description**
Notebook server images are really old and need to be updated to the latest versions of packages available.

**Comments**
* Discussed in notebook WG call on Sep 28, 2023

### PR DESCRIPTION
**Pull request by mishraprafful**
### Related Issues/PRs
Closes #7313 

### Proposed changes 
* Update all notebook images with the latest packages.

### Comments
Images builds were tested with docker on a local workstation.

### CODE REVIEW

**Comment by kimwnasptd** [comment]
@mishraprafful if you have the cycles let's discuss about implementing CI workflows for building the images in each PR.

File: components/example-notebook-servers/jupyter-pytorch-full/requirements.txt [hunk 1]
```diff
@@ -1,23 +1,23 @@
 # kubeflow packages
-kfp==1.6.3
-kfp-server-api==1.6.0
-kfserving==0.5.1
+kfp==2.3.0
+kfp-server-api==2.0.1
+kfserving==0.6.1
```
Comment (kimwnasptd) [inline components/example-notebook-servers/jupyter-pytorch-full/requirements.txt:4]: Should we drop this package and install the corresponding kserve package instead?

**Comment by thesuperzapper** [comment]
/hold

@mishraprafful sorry to do this, but we have had to make a separate PR that updates the images and fixes a number of issues like adding ARM support and properly updating CUDA/TF/Pytorch.

Please don't be disheartened, I will post the new PR soon so you can see.

File: components/example-notebook-servers/jupyter-pytorch-full/requirements.txt (hunk 1 above)
Comment (juliusvonkohout) [reply components/example-notebook-servers/jupyter-pytorch-full/requirements.txt:4]: yes 0.11.1 from here https://pypi.org/project/kserve/

**Comment by juliusvonkohout** [comment]
@thesuperzapper if its in the past, then i am wondering where the changes are.

**Comment by thesuperzapper** [comment]
@juliusvonkohout @mishraprafful the new PR (which includes ARM support) is https://github.com/kubeflow/kubeflow/pull/7357.

I will close this one for now, and we should be able to get the new one merged very quickly, once @kimwnasptd wakes up!

/close

**Pull request closed without merging**